ansible/roles-infra/infra-images/defaults/main.yaml

```diff
@@ -1,3 +1,86 @@
 ---
 # by default use the agnosticd 'instances' variable
 infra_images_instances: "{{ instances | default([]) }}"
+
+
+# Predefined search dicts for commonly used images
+infra_images_redhat_owner_id: 309956199498
```

Comment:
Should this also be a var, perhaps defaulted to `309956199498`?
And does it need to be extended in some way as it is very AWS/RedHat centric or is the assumption this is only a Red Hat thing? The role is generic or should it be an AWS-specific role?

`infra_images_redhat_owner_id: "{{ infra_images_owner | default('309956199498' }}"`

Reply:
that variable is local to the default.yaml file, i don't expect it to be used outside of it.

It's just a convenience var, but i could have use the id all over the place when defining the infra_images_predefined mapping